reflex/reflex.py

```diff
@@ -124,13 +129,20 @@ def run(
         )
         raise typer.Exit()
 
+    # Update db migration settings
+    if not db_migrate:
+        get_config().db_migrate = False
```

Comment:
Maybe we don't need the `db_migrate` in the config anymore. Just have them explicitly do migrations.